B827EBFFFEEA79DF.json

```diff
@@ -0,0 +1,24 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFEEA79DF",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      },
+      {
+        "server_address": "log.gatewaystats.org",
```

Comment:
I believe this service does not work anymore, so you can remove it from the servers list, but I will merge anyway and you can send another PR to remove if you want.